app-misc/ddcutil/ddcutil-0.9.1.ebuild

```diff
@@ -27,9 +27,17 @@ RDEPEND="dev-libs/glib:2
 	X? (
 		x11-libs/libXrandr
 		x11-libs/libX11
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	usb-monitor? (
```

Comment:
Should `usb-monitor` be inside the `user-permissions` block of RDEPEND? Otherwise `USE='usb-monitor -user-permissions'` ends up adding the `video` group now when the old code did not trigger `enewgroup video` in that case.

Is `usb-monitor` even meaningful without `user-permissions`? If it's not meaningful, probably also add a `REQUIRED_USE` to block the case of `USE='usb-monitor -user-permissions'`

Reply:
Thanks for the review!

The way I understand https://www.ddcutil.com/usb/, `usb-monitor -user-permissions` may make sense, since that would still allow managing such monitor by user that is in video group. I.e. if a monitor is managed through USB, there may not be a need to manage it through I2C. Sadly, none of the monitors I have work with USB, so I cannot test this.

I do agree that the ebuild in the form that I submitted would exhibit inconsistent behavior in regards to those useflags. 

If my understanding above is correct, I could flatten the nested conditionals and make the checks independent.

I would also like to clarify, should I add revisions to the updated versions of ddcutil ebuilds?